### PR TITLE
Add aria-attributes to slider component

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3226 as per the [spec](https://www.w3.org/TR/wai-aria-practices/#x3-18-slider)

- aria-valuemin, valuemax, valuenow
- aria-labels
- tabindex="0"

## QA

- Pull code
- Run `./run`
- Verify the added attributes are correct

### CODE REVIEW

File: templates/docs/examples/patterns/slider/slider-input.html [hunk 1]
```diff
@@ -1,16 +1,40 @@
-{% extends "_layouts/examples.html" %}
-{% block title %}Slider / Input{% endblock %}
-
-{% block standalone_css %}patterns_slider{% endblock %}
-
-{% block content %}
-<div class="p-slider__wrapper">
-  <input class="p-slider" type="range" min="0" max="100" value="50" step="1" id="slider3">
-  <input class="p-slider__input" type="text" maxlength="3" id="slider3-input">
+{% extends "_layouts/examples.html" %} {% block title %}Slider / Input{% endblock %} {% block standalone_css %}patterns_slider{% endblock %} {% block content %} <div class="p-slider__wrapper">
```
Comment (bartaz) [inline templates/docs/examples/patterns/slider/slider-input.html:1]: Prettier messed up the template formatting.
Could you get the top part template tags back to original formatting (one per line)?

File: templates/docs/examples/patterns/slider/slider-input.html [hunk 2]
```diff
@@ -1,16 +1,40 @@
...
+  <input class="p-slider"
+    type="range"
+    aria-value-min="0"
```
Comment (bartaz) [inline templates/docs/examples/patterns/slider/slider-input.html:4]: Is this needed when there is `min` defined?
Seems like a lot of duplication to keep all values in 2 places...

Comment (lyubomir-popov) [reply templates/docs/examples/patterns/slider/slider-input.html:4]: I couldn't find explicit informaitno that it is not needed, and saw an example at mdn that hade both (link in the attached issue) so figured it is the safer approach...

**Review by bartaz — APPROVED** [review]

**Review by bartaz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**